.github/workflows/build.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
       - name: Build and push
-        uses: docker/build-push-action@v5.0.0
+        uses: docker/build-push-action@v5.1.0
```

Comment:
Rather than going to `v5.1.0`, shouldn't this have been generalized to `v5`?

Ditto line 64.

Reply:
This seems to just be Renovate behavior. We removed digest pinning, but it then moves to .z pinning instead.

Reply:
I'm guessing that, because we have `v5.0.0` in the file and there is an update available, the bot proposed changing the file to `v5.1.0` (which is all good and desirable).

My question is, if we instead had `v5` in the file, would the workflow work (I expect that it would) and would the bot leave the line unchanged (I expect that it would).  This would decrease the amount of noise that we experience, which would be better.